docs/resources/runbook.md

```diff
@@ -50,6 +70,7 @@ resource "firehydrant_runbook" "example-runbook" {
 The following arguments are supported:
 
 * `name` - (Required) The name of the runbook.
+* `attachment_rule` - (Optional) JSON string representing the attachment rule configuration for the runbook.
```

Comment:
nit: could this be moved down so that it is at the top of the optional args? i'm trying to keep everything following the pattern of required attributes first in alphabetical order and then optional attributes in alphabetical order

Reply:
also, i think it might be worth documenting that this defaults to attaching automatically if attachment rule is not specified.